scripts/Muon/view_constructor.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import (absolute_import, division, print_function)
+
+from Muon import FFT_view
+from Muon import MaxEnt_view
+from Muon import transform_selection_view
+
+"""
+ simple class to create a single object
+ containing all of the views.
+ Only need to pass a single object to all
+ presenters
+"""
```

Comment:
The docstring should be moved to the line below `class ViewConstructor(object):`